library/Fission/Types.hs

```diff
@@ -563,3 +568,29 @@ instance Domain.Creator Fission where
   create domainName userId zoneId now =
     runDB $ Domain.create domainName userId zoneId now
 
+instance Challenge.Creator Fission where
+  create email = 
+    runDB $ Challenge.create email
+
+instance Challenge.Verifier Fission where
+  verify challenge = 
+    runDB $ Challenge.verify challenge
+
+instance Mail.MonadMail Fission where
+  sendVerifyEmail recipient (Challenge hash) = do
+    httpManager <- liftIO $ HTTP.newManager HTTP.tlsManagerSettings
```

Comment:
atleast testing locally, the `httpManager` is not TLS enabled. And looking at `server.Main.hs`, it looks like it's never TLS enabled

Do we need a second manager in `Config`? or is it fine to create one ephemerally here?

Reply:
Our server doesn't use TLS directly anymore because we're using the AWS proxy for certs

Reply:
> Do we need a second manager in Config? or is it fine to create one ephemerally here?

If we're going to be making TLS requests, then yes, we should have one kicking around at all times. It takes up very little room in memory, and can get reused for lots of things. Plus it would be good to avoid the repeated setup/teardown.

Reply:
> Our server doesn't use TLS directly anymore because we're using the AWS proxy for certs
right right :ok_hand: 

> we should have one kicking around at all times in Config
ok cool. I'll move it to Config :+1: